.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "prettier.singleQuote": true
```

Comment:
Is it a style convention or for performance purposes?

Reply:
In general .vscode folder can contain extentions configuration, scripts, ect. It's a project decided to keep .vscode folder in git or add it into .gitignore

And in terms of prettier, in web projects it's common to use it for autoformatting based on defined project style convention.